### PR TITLE
fix: card design revisions v1

### PR DESCRIPTION
**Pull request by davidmenendez**
Contributes to #380 

handles all the feedback listed here https://github.com/carbon-design-system/ibm-cloud-cognitive/issues/380#issuecomment-788403005

#### What did you change?

Updates to the `Card` component

#### How did you test and verify your work?

Run the storybook and verify the changes align with the design feedback and doesn't break anything

### CODE REVIEW

File: packages/experimental/src/components/Card/_storybook-styles.scss [hunk 1]
```diff
@@ -1,8 +1,45 @@
...
+.media-ratio-43 .media-box {
+  padding-bottom: 75%;
+}
+
+.media-ratio-32 .media-box {
+  padding-bottom: 66.66%;
+}
+
+.media-ratio-169 .media-box {
+  padding-bottom: 56.25%;
```
Comment (matthewgallo) [inline packages/experimental/src/components/Card/_storybook-styles.scss:34]: I wonder if we could use [`aspect-ratio`](https://developer.mozilla.org/en-US/docs/Web/CSS/aspect-ratio) here?

Comment (lee-chase) [reply packages/experimental/src/components/Card/_storybook-styles.scss:34]: Or even https://react.carbondesignsystem.com/?path=/story/aspectratio--playground

Comment (davidmenendez) [reply packages/experimental/src/components/Card/_storybook-styles.scss:34]: oh wow i can't believe they already had a component for this built. imagine my surprise. i'll go ahead and see if I can use either of these to improve this. thanks!

**Comment by davidmenendez** [comment]
@lee-chase @matthewgallo thanks for the feedback. i was able to use the carbon `AspectRatio` component. It really cleaned everything up. Usage of it in the `media left` story is still tricky, because there needs to be some sort of defined width, so I still had to leave a set width.

**Review by matthewgallo — APPROVED** [review]
Looks good! One small thing I noticed was that there are not any focus states for the `with action icon` and `clickable card with onclick` stories. But I'm guessing they're coming in a future PR

**Pull request merged**